lib/testing_appserver/appserver_test_instance.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'active_support'
+require 'httparty'
+require 'onlyoffice_testrail_wrapper'
+require 'onlyoffice_webdriver_wrapper'
+require 'onlyoffice_tcm_helper'
+require 'palladium'
+require_relative 'data/user_data/user_data'
+require_relative 'data/user_data/default_user_data'
+require_relative 'appserver_helper'
+
+include OnlyofficeTestrailWrapper
+include OnlyofficeWebdriverWrapper
+
+module TestingAppServer
+  # Instance of browser to perform actions
+  class AppserverTestInstance
+    attr_accessor :webdriver
+    alias driver webdriver
```

Comment:
I think was aliases was used for historical reason, we can leave only `webdriver` name and don't use others